doc/release-notes-1.3.0.rst

```diff
@@ -0,0 +1,577 @@
+.. _ncs_release_notes_130:
+
+|NCS| v1.3.0 Release Notes
+##########################
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products.
+It includes the MCUboot and the Zephyr RTOS open source projects, which are continuously integrated and re-distributed with the SDK.
+
+The |NCS| is where you begin building low-power wireless applications with Nordic Semiconductor nRF52, nRF53, and nRF91 Series devices.
+nRF53 Series devices (which are pre-production) and Thread, Zigbee, and Bluetooth Mesh protocols are supported for development in v1.3.0 for prototyping and evaluation.
+Support for production and deployment in end products is coming soon.
+
+
+Highlights
+**********
+
+* Support for Thread and Zigbee
```

Comment:
```
nRF Desktop released
```

Reply:
@jhn-nordic

Reply:
"nRF Desktop released" looks good to me.

Reply:
"released" sounds a bit confusing in this context (everything in release notes is released...). "Support for nRF Desktop" or "nRFDesktop supported for production" or "nRFDesktop supported for development" is good - take your pick, @pdunaj :)